pyproject.toml

```diff
@@ -23,25 +23,85 @@ known_first_party = [
     "tests",
 ]
 skip_glob = [
+
+
```

Comment:
```suggestion
```

Reply:
not sure, so rather take the safe side :]